test/server/triggers.ts

```diff
@@ -0,0 +1,75 @@
+import { expect, test } from 'vitest'
+import { app } from './utils'
+
+test('trigger list filtering', async () => {
+  const res = await app.inject({
+    method: 'GET',
+    path: '/triggers?limit=5',
+  })
+  expect(res.statusCode).toBe(200)
+  const triggers = res.json()
+  expect(Array.isArray(triggers)).toBe(true)
+  expect(triggers.length).toBeLessThanOrEqual(5)
+})
+
+test('trigger list with specific included schema', async () => {
```

Comment:
we need to create some in public and in other schema(s) first (maybe we have one from migrations in public)